pyproject.toml

```diff
@@ -29,16 +29,16 @@ plugins = [
 [tool.pyright]
 include = ["snapcraft", "tests"]
 exclude = ["build", "tests/legacy", "tests/spread"]
-pythonVersion = "3.10"
+pythonVersion = "3.12"
 
 [tool.pytest.ini_options]
 minversion = "7.0"
-required_plugins = ["pytest-cov>=4.0", "pytest-mock>=3.10", "pytest-subprocess>=1.4"]
+required_plugins = ["pytest-cov>=4.0", "pytest-mock>=3.12", "pytest-subprocess>=1.4"]
```

Comment:
I was going to say this was changed by mistake but apparently pytest-mock 3.12 adds support for python 3.12, heh